lib/src/chart/radar_chart/radar_chart_data.dart

```diff
@@ -0,0 +1,267 @@
+import 'dart:ui';
+
+import 'package:equatable/equatable.dart';
+import 'package:fl_chart/fl_chart.dart';
+import 'package:fl_chart/src/utils/lerp.dart';
+import 'package:flutter/foundation.dart';
+import 'package:flutter/material.dart';
+
+typedef GetTitleByIndexFunction = String Function(int index);
+
+class RadarChartData extends BaseChartData {
+  //ToDo(payam) : document data classes
+  RadarChartData({
+    this.dataSets = const [],
+    this.fillColor = Colors.transparent,
+    this.chartBorderData = const BorderSide(color: Colors.black, width: 2),
```

Comment:
And for this, I suggest `circularBorderData` (to prevent confusion with `borderData`)

Reply:
@imaNNeoFighT   as I mentioned above the radar chart is not always circular. so I decided to rename this field to radarBackgroundColor and radarBorderData. If you have any other suggestions please let me know.